src/components/NavBar/NavBar.js

```diff
@@ -29,16 +29,16 @@ const NavBar = () => {
 
   return (
     <div className="navbar-container">
-      <div className="logo-container">
-        <Link to="/">
-          <img className="bw-logo" alt="blue-witness-logo" src={bwlogo} />
-        </Link>
+      <div className="menu-pusher">
         <a
-          href="https://www.humanrightsfirst.org"
+          href="https://humanrightsfirst.org"
           target="_blank"
           rel="noopener noreferrer"
         >
-          <img src={hrflogo} className="hrf-logo" alt="HRF logo" />
+          <div className="logo-container">
+            <img className="bw-logo" alt="blue-witness-logo" src={bwlogo} />
+            <img src={hrflogo} className="hrf-logo" alt="HRF logo" />
+          </div>
         </a>
       </div>
```

Comment:
I see that you consolidated the images into one div and made that div the link to the HRF website. What happens if a mobile or smaller screen user is on another part of the site and wants to return to the map? Intuitively, it seems that the Blue Witness logo would be a link to back to the homepage but now it will take them to HRF? From a user perspective this seems less than ideal.

Reply:
I was kinda on the fence here. I think if we're getting users coming here on their own, then yes it would be better if the logo acts as a link to home. I was kinda imagining that users would come here for research from the hrf site, and that they might be using hrf as a larger source here. But I can definitely change this.